controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -106,6 +110,14 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 	// start to update mco status
 	StartStatusUpdate(r.Client, instance)
 
+	if os.Getenv("UNIT_TEST") != "true" {
+		// start placement controller
+		placemengctrl.StartPlacementController(r.Manager)
+
+		// setup ocm addon manager
+		certctrl.Start(r.Client)
+	}
```

Comment:
They will be started many times in reconciler w/o control

Reply:
fixed